servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/tests/BaseTestNessieApi.java

```diff
@@ -521,6 +522,57 @@ public void commitMergeTransplant() throws Exception {
         .containsKeys(ContentKey.of("a", "a"), ContentKey.of("b", "a"), ContentKey.of("b", "b"));
   }
 
+  @Test
+  public void mergeTransplantDryRunWithConflicts() throws Exception {
```

Comment:
Guess this test should better live in `org.projectnessie.versioned.tests.AbstractMerge`

Reply:
Do we not want end-to-end validation for `dryRun` handling? I did not see any other API-level test that exercised `dryRun` parameters.

Reply:
Hm - then maybe have it in both places. A simple one in `o.p.jaxrs.tests` validating `dryRun` and the functional one in `o.p.versioned.tests`.

The benefit of having it in `org.projectnessie.versioned.tests` is that it's easily debuggable w/o the REST overhead (and thread "jumps").

Reply:
will do

Reply:
Does #6685 cover what you meant, @snazy ?

Reply:
That's more than I thought of - but great stuff :)